tests/test_settings.py

```diff
@@ -0,0 +1,99 @@
+import os
+from unittest import mock
+
+import pytest
+
+from screenpy import settings as screenpy_settings
+from screenpy.narration.stdout_adapter import settings as stdout_adapter_settings
+from screenpy.configuration import ScreenPySettings
+from screenpy.narration.stdout_adapter.configuration import StdOutAdapterSettings
+
+
+class TestSettings:
+    def test_pyproject_overwrites_initial(self):
+        mock_open = mock.mock_open(read_data=b"[tool.screenpy]\nTIMEOUT = 500")
+
+        with mock.patch("pathlib.Path.open", mock_open):
+            settings = ScreenPySettings()
+
+        assert settings.TIMEOUT == 500
+
+    def test_env_overwrites_pyproject(self):
+        mock_open = mock.mock_open(read_data=b"[tool.screenpy]\nTIMEOUT = 500")
+        mock_env = {"SCREENPY_TIMEOUT": "1337"}
+
+        with mock.patch("pathlib.Path.open", mock_open):
+            with mock.patch.dict(os.environ, mock_env):
+                settings = ScreenPySettings()
+
+        assert settings.TIMEOUT == 1337
+
+    def test_init_overwrites_env(self):
+        mock_env = {"SCREENPY_TIMEOUT": "1337"}
+
+        with mock.patch.dict(os.environ, mock_env):
+            settings = ScreenPySettings(TIMEOUT=9001)
+
+        assert settings.TIMEOUT == 9001
+
+    def test_can_be_changed_at_runtime(self):
+        try:
+            screenpy_settings.TIMEOUT = 4
+        except TypeError:
+            msg = "ScreenPySettings could not be changed at runtime."
+            raise AssertionError(msg)
```

Comment:
This has the possibility passing incorrectly.  If a TypeError were to be raised but the message didn't match, the test wouldn't fail.  
I'm guessing this could be simplified to simply 
```python
def test_can_be_changed_at_runtime(self):
    screenpy_settings.TIMEOUT = 4
```

Reply:
I think you're misunderstanding what this test is doing—it's not asserting the message is correct, it's changing a `TypeError` (what Pydantic raises when you try to change an immutable Settings class) into an `AssertionError` and supplying a message, meaning this test has failed.